### ISSUE TITLE
Biographical sketch should appear as chapter-level heading in the ToC

### ISSUE THREAD

**Issue by piyanatk** [GarenSidonius/ASU-Dissertation-Template#17]
A biographical sketch should appear as a chapter-level heading in the ToC per ASU formatting team. I got this to work by inserting a biographical sketch with a `*` option and manually adding a heading to the ToC.

```latex
\chapter*{Biographical Sketch}          %~Biographical Sketch is optional
\input{biography}                       %<Enter the name of the .tex file containing your
                                        %   biography or omit this line and type in
                                        %   your biography here (1 paragraph)

\phantomsection                         % Need for hyperref 
\addcontentsline{toc}{chapter}{%        % Add a chapter-level heading for 
  \hspace{-\cftchapterindent}%          %  a biographical sketch to the ToC
  Biographical Sketch%
}
```
However, a user will also have to comment/uncomment parts of the codes that do the mannual ToC heading insertion.

Is there a better way to implement this? Perhaps, forcing the heading to be added to chapter-level before inserting a biography with `\chapter{}`?

**Comment by GarenSidonius** [comment]
I've been thinking about this one, too, especially how to include this optionally for the user without them having to uncomment so many lines. Still thinking ...

**Comment by GarenSidonius** [comment]
This issue is resolved in 94734a9dc594339ef458df5f729e2cb058dbcb68.
